source/configure/environment-configuration-settings.rst

```diff
@@ -4134,6 +4134,58 @@ Allow untrusted internal connections
 |   but not ``10.0.16.20``.                                                                                                                     |
 +-----------------------------------------------+-----------------------------------------------------------------------------------------------+
 
+Mobile Security
+---------------
+
+.. config:setting:: mobile-security-enabled
+  :displayname: Enable Biometric Authentication
+  :systemconsole: Environment > Mobile Security
+  :configjson: .NativeAppSettings.MobileEnableBiometrics
+  :environment: MM_NATIVEAPPSETTINGS_MOBILEENABLEBIOMETRICS
+  :description: Enforces biometric authentication (with PIN/passcode fallback) before accessing the app. Users will be prompted based on session activity and server switching rules.
+
+    - **true**: Biometric authentication is enabled.
+    - **false**: **(Default)** Biometric authentication is disabled.
+
+
+Enable Biometric Authentication
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. include:: ../_static/badges/ent-only.rst
+  :start-after: :nosearch:
+
+.. config:setting:: mobile-security-enabled
+  :displayname: Enable Jailbreak/Root Protection
+  :systemconsole: Environment > Mobile Security
+  :configjson: .NativeAppSettings.MobileJailbreakProtection
+  :environment: MM_NATIVEAPPSETTINGS_MOBILEJAILBREAKPROTECTION
+  :description: Prevents access to the app on devices detected as jailbroken or rooted. If a device fails the security check, users will be denied access or prompted to switch to a compliant server.
+
+    - **true**: Jailbreak/Root protection is enabled.
+    - **false**: **(Default)** Jailbreak/Root protection is disabled.
+
+Enable Jailbreak/Root Protection
```

Comment:
there are some caveats to this that can be referenced [here](https://docs.expo.dev/versions/latest/sdk/device/#deviceisrootedexperimentalasync) not sure if we want to add that in here

Reply:
@cwarnermm You added this caveat here. Do you think it is also needed in the in-product documentation (i.e. the help text for this setting)?

Reply:
@larkox - If the intention is to include a link in the System Console to the config docs, then that's likely sufficient for the caveat-level details. An admin could visit the docs, see the caveat link, learn more, and make an informed decision.

In my mind, the caveats are good context, but not necessarily make-or-break details. Thoughts?